### PR TITLE
Use activemq-bom

### PR DESCRIPTION
**Pull request by eddumelendez**
ActiveMQ 6.1.3 provides activemq-bom.

### CODE REVIEW

File: spring-boot-project/spring-boot-dependencies/build.gradle [hunk 1]
```diff
@@ -14,37 +14,10 @@ bom {
...
-	library("ActiveMQ", "6.1.2") {
+	library("ActiveMQ", "6.1.3") {
 		group("org.apache.activemq") {
-			modules = [
-					"activemq-amqp",
-					"activemq-blueprint",
-					"activemq-broker",
-					"activemq-client",
-					"activemq-console" {
-						exclude group: "commons-logging", module: "commons-logging"
```
Comment (snicoll) [inline spring-boot-project/spring-boot-dependencies/build.gradle:25]: Did you check this is no longer necessary?

File: spring-boot-project/spring-boot-dependencies/build.gradle [hunk 2]
```diff
@@ -14,37 +14,10 @@ bom {
...
-					"activemq-openwire-generator",
-					"activemq-openwire-legacy",
-					"activemq-osgi",
-					"activemq-pool",
-					"activemq-ra",
-					"activemq-run",
-					"activemq-runtime-config",
-					"activemq-shiro",
-					"activemq-spring" {
-						exclude group: "commons-logging", module: "commons-logging"
```
Comment (snicoll) [inline spring-boot-project/spring-boot-dependencies/build.gradle:44]: Ditto

**Review by snicoll — COMMENTED** [review]
Thanks for the PR @eddumelendez but there are two excludes there that are lost with the use of the BOM. Did you check that it isn't relevant anymore?

File: spring-boot-project/spring-boot-dependencies/build.gradle (hunk 1 above)
Comment (eddumelendez) [reply spring-boot-project/spring-boot-dependencies/build.gradle:25]: Hi @snicoll, those are still necessary. How should I proceed?

Comment (wilkinsona) [reply spring-boot-project/spring-boot-dependencies/build.gradle:25]: We can use the bom and then also declare the modules that require exclusions. Here's an example for Micrometer:

```
	library("Micrometer", "1.9.17") {
		considerSnapshots()
		group("io.micrometer") {
			modules = [
				"micrometer-registry-stackdriver" {
					exclude group: "javax.annotation", module: "javax.annotation-api"
				}
			]
			imports = [
				"micrometer-bom"
			]
		}
	}
```

This uses `micrometer-bom` while also adding an exclusion to the `micrometer-registry-stackdriver` dependency that it manages.

Comment (eddumelendez) [reply spring-boot-project/spring-boot-dependencies/build.gradle:25]: Thanks!

**Comment by eddumelendez** [comment]
PR has been updated

**Comment by wilkinsona** [comment]
Thanks very much, @eddumelendez.

**Pull request closed without merging**